examples/integrations/oci/oci-secrets-mp-config-source/README.md

```diff
@@ -0,0 +1,331 @@
+
+    1. On the left side of the page, in the Resources section, click on Secrets.
+
+    2. Press the **Create Secret** button.
+
+    3. Supply a name for the secret in the Name text field. This example uses "helidon-examples-secret", but it can be any
+       name you like that conform's to OCI's valid character requirements.
+
+    4. Supply a description for the secret in the Description text field. This example uses "A secret for Helidon
+       Examples" but it can be any text you like.
+
+    5. Choose the Helidon Examples Master Encryption Key in the Encryption Key drop-down list (the master encryption key
+       you just created).
+
+    6. Supply a plain text value for the secret in the Secret Contents text area. This example uses "secret value", but it
+       can be any text you like.
+
+    7. Press the **Create Secret** button toward the bottom of the page.
+
+9. Create a policy to allow your new user in your new group to read your new secret in your new vault.
+
+    1. Go to https://cloud.oracle.com/identity/policies/ (open this link in a new browser tab or window).
+
+    2. Press the **Create Policy** button.
+
+    3. Supply a name for this new policy in the Name text field. This example uses
+       "helidon-examples-group-manipulate-secrets", but it can be any name you like that conforms to OCI's valid
+       character requirements.
+
+    4. Supply a description for this new policy in the Description text field. This example uses "Permits
+       helidon-examples-group members to manipulate secrets in the helidon-examples compartment" but it can be any text
+       you like.
+
+    5. Select your new compartment from the Compartment drop-down list.
+
+    6. In the **Policy Builder** area, select "Key and Secret Management" from the Policy use cases drop-down list.
+
+    7. In the **Policy Builder** area, select "Let users read, update and rotate all secrets" from the Common policy
+       template drop-down list.
+
+    8. In the **Policy Builder** area, select your new group from the Groups drop-down list.
+
+    9. In the **Policy Builder** area, select your new compartment from the Location drop-down list.
+
+    10. Observe that the **Policy Statements** area now contains a statement like "Allow group
+        **helidon-examples-group** to use secret-family in **compartment helidon-examples**".
+
+    11. Press the **Create** button toward the bottom of the page.
+
+To recap, you have, in dependency order:
+
+* Created a user ("helidon-examples")
+
+* Created a `${HOME}/.oci/config` file with that user's authentication information
+
+* Created a group ("helidon-examples-group")
+
+* Added the user to the group
+
+* Created a compartment ("helidon-examples")
+
+* Created a vault in the compartment ("helidon-examples-vault")
+
+* Created a secret in the vault ("helidon-examples-secret")
+
+* Created a policy that lets your group's users manipulate secrets found in your compartment
```

Comment:
It would be good to include the literal policy statement here.

Reply:
Sure; why?

Reply:
Ah, wait, I [already did this](https://github.com/helidon-io/helidon/blob/32c2976ad83628dbacb3ac4df5d7e85b709b4eb1/examples/integrations/oci/oci-secrets-mp-config-source/README.md#:~:text=Observe%20that%20the%20Policy%20Statements%20area%20now%20contains%20a%20statement%20like%20%22Allow%20group%20helidon%2Dexamples%2Dgroup%20to%20use%20secret%2Dfamily%20in%20compartment%20helidon%2Dexamples%22.).